### DISCUSSION TITLE
Docs Dagger Personas

### DISCUSSION THREAD

**Discussion by marcosnils** [dagger/dagger#4550] [Ideas]
Speaking with @vikram-dagger about docs next steps as highlighted here (https://github.com/dagger/dagger/discussions/4213), a topic came up about our docs target user/persona not being clear to everyone which makes it difficult for us to optimize our docs experience and effectively measure their goal.

We briefly dug into it and in a first pass we generalized what we think our users could be into 4 personas: 

1. DevOps engineers that suffer the pain of maintaining large CI/CD pipelines in their organizations and are looking forward to replace them with something else
2. Infra / Platform / DevOps users that are switching CI's (i.e: people moving away from CircleCI due to their recent breach) and need to understand how/if Dagger can help here
3. DevOps / Dev enthusiasts that are attracted by the "programmability" aspects of Dagger and are interested in the development aspects mostly to build things on top of it.
4. Users that don't have a lot of experience with containers / docker and are jumping in the DevOps / Platform Engineer wagon.

Questions for everyone: 

1. Are we on the right track with these assumptions? Shall we remove/add another type of user?
2. Should we pic X types of personas and think about building different docs paths and conversion objectives for each of these?
3. Seems like our current website / docs is still not evidently clear for everyone (https://discord.com/channels/707636530424053791/708371226174685314/1072158202352902164). Is it worth to invest time refining this before having a clear product to offer?

cc @vikram-dagger @jpadams @shykes @samalba

**Comment by mircubed** [comment]
This is something that I want to pull @d3rp3tt3 asap into when she gets on board :)

**Comment by crjm** [comment]
As discussed with @vikram-dagger, we are going to find the necessary information to validate the different personas and adjust their characteristics/add new ones, depending on what we find. 
After doing that, we should add to the docs different user learning paths, one for each persona.
I'll create actionable items, as I'm sure we can gather some conclusions with the data we already have.
